### PR TITLE
Cpl 04 n child

### PR DESCRIPTION
**Pull request by TamoorKhokhar**

### CODE REVIEW

File: src/constants/appConstants.js [hunk 1]
```diff
@@ -1,2 +1,15 @@
...
+export const lunchStart = 0 * 60 + 0;
+export const lunchEnd = 13 * 60 + 0;
+export const eveningTeaStart = 0 * 60 + 0;
+export const eveningTeaEnd = 17 * 60 + 0;
+export const morningResultStart = 11 * 60 + 0;
+export const morningResultEnd = 23 * 60 + 0;
+export const lunchResultStart = 13 * 60 + 0;
+export const lunchResultEnd = 23 * 60 + 0;
+export const eveningResultStart = 17 * 60 + 0;
+export const eveningResultEnd = 23 * 60 + 0;
```
Comment (MuhAhs001) [inline src/constants/appConstants.js:14]: plus 0 is really effecting?

File: src/constants/appConstants.js [hunk 2]
```diff
@@ -1,2 +1,15 @@
...
+
+export const morningTeaStart = 0 * 60 + 0;
+export const morningTeaEnd = 11 * 60 + 0;
+export const lunchStart = 0 * 60 + 0;
+export const lunchEnd = 13 * 60 + 0;
+export const eveningTeaStart = 0 * 60 + 0;
+export const eveningTeaEnd = 17 * 60 + 0;
+export const morningResultStart = 11 * 60 + 0;
+export const morningResultEnd = 23 * 60 + 0;
+export const lunchResultStart = 13 * 60 + 0;
```
Comment (MuhAhs001) [inline src/constants/appConstants.js]: lunchResultStart
lunchEnd 
having same value why two different constants

File: src/utils/services/tableDataServices.js [hunk 3]
```diff
@@ -0,0 +1,73 @@
+import { request, setHeaders } from './commonServices';
+import axios from 'axios';
+
+export const signUp = async (payload) => {
+  try {
+    return await axios.post('https://lu-meal-stage.herokuapp.com/api/users/sign-up', payload);
+  } catch (error) {
```
Comment (MuhAhs001) [inline src/utils/services/tableDataServices.js:8]: move to constants

File: src/utils/services/tableDataServices.js [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+    return error;
+  }
+};
+export const signIn = async (body) => {
+  try {
+    const response = await request('/api/users/log-in', 'POST', body);
```
Comment (MuhAhs001) [inline src/utils/services/tableDataServices.js:14]: same for all other

File: src/constants/appConstants.js (hunk 1 above)
Comment (TamoorKhokhar) [reply src/constants/appConstants.js:14]: A complete Format 0 is used bcz when we have to change time in next phase like 30 minutes so we have to replace 30 with 0

**Review by MuhAhs001 — APPROVED** [review]

**Pull request merged**